xblockutils/resources.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+
+import pkg_resources
+from django.template import Context, Template
+
+
+class ResourceLoader(object):
+    """Loads resources relative to the module named by the module_name parameter."""
+    def __init__(self, module_name):
+        self.module_name = module_name
+
+    def load_unicode(self, resource_path):
+        """
+        Gets the content of a resource
+        """
+        resource_content = pkg_resources.resource_string(self.module_name, resource_path)
+        return unicode(resource_content)
```

Comment:
`unicode(string)` is not a good thing to do. Either string is already unicode, in which case it's a no-op, or you will end up decoding it as ascii, which isn't as good as decoding it as utf-8.